modules/puzzles/module.py

```diff
@@ -60,14 +59,44 @@ def coordinates(self, numbers):
         """
         Geeft alle mogelijke locaties op de campus met coordinaten bestaande uit de gegeven getallen. Getallen dienen te worden gescheiden met komma's
         """
-        results = util.brute_force_coordinates(numbers.split(','))
-        self.respond('Er zijn %i locaties op de campus met deze getallen.' % (len(results)))
-        if(results):
-            if len(results) <= 5:
-                for result in results:
-                    self.send_location(result[0], result[1])
-            elif len(results) <= 15:
+        number_list = numbers.split(',')
+        if all([x.isdigit() for x in number_list]):
+            results = util.brute_force_coordinates(number_list)
+            self.respond('Er zijn %i locaties op de campus met deze getallen.' % (len(results)))
+            if(results):
                 self.respond('\n'.join(['%s, %s' % x for x in results]))
+        else:
+            self.respond('Verkeerde input. Ik verwacht bijvoorbeeld \'/coordinates 52,248,219,6,85,40,38\'.')
+
+    @command
+    def ascii(self, numbers):
+        """
+        Vertaalt de waarden naar ASCII-tekens. ASCII-waarden van letters liggen tussen de 65 en de 121. De inputwaarden dienen gescheiden te worden met komma's.
+        """
+        number_list = numbers.split(',')
```

Comment:
Ik heb gister de PR van rens gemerged die varargs toestaat. Dat betekent dat als je als argument aangeeft meerdere argumenten te kunnen ontvangen, het framework op spaties split. Je code zou er dan zo uitzien:

```
@command
def ascii(self, *number_list):
```

Reply:
Dit heb ik geïmplementeerd maar ik kwam tot de conclusie dat het toch fijner is om er komma's tussen te zetten. Het is evenveel werk en handiger met meerdere soorten argumenten